apps/testing/src/main/java/com/akto/testing/Main.java

```diff
@@ -197,6 +200,61 @@ public static void main(String[] args) throws InterruptedException {
                         Filters.eq("_id", testingRun.getId()),  completedUpdate
                 );
 
+                if(summaryId != null && testingRun.getTestIdConfig() != 1){
```

Comment:
do not keep whole logic in main.java, extract it out in a function and call that method here